fjord/mailinglist/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+
+from fjord.base.models import ModelBase
+
+
+class MailingList(ModelBase):
+    name = models.CharField(
+        max_length=100, unique=True,
+        help_text=u'Unique name to identify the mailing list'
+    )
+    members = models.TextField(
+        blank=True, default=u'',
+        help_text=(
+            u'List of email addresses for the list--one per line. You '
```

Comment:
Someday someone will write a version of `dedent` that also joins the lines together. And on that day, we will rejoice.

But not today!